search-commons/src/main/java/no/unit/nva/search2/common/Query.java

```diff
@@ -123,7 +123,7 @@ public Stream<QueryContentWrapper> createQueryBuilderStream() {
 
         getSortStream().forEach(builder::sort);
 
-        if (getMediaType().is(JSON_UTF_8)) {
+        if (getMediaType().is(JSON_UTF_8) && isDefined(ALL)) {
```

Comment:
ups, should have been here, but it is harmless, (it will not go through all aggregations code, if 'all' is not supplied)